src/components/Swap/Toolbar/Caption.tsx

```diff
@@ -1,12 +1,11 @@
 import { Trans } from '@lingui/macro'
 import { Placement } from '@popperjs/core'
+import { formatPriceImpact } from '@uniswap/conedison/format'
 import { Currency, CurrencyAmount, Token } from '@uniswap/sdk-core'
-import Column from 'components/Column'
 import Row from 'components/Row'
-import Rule from 'components/Rule'
 import Tooltip from 'components/Tooltip'
 import { loadingCss } from 'css/loading'
-import { PriceImpact } from 'hooks/usePriceImpact'
+import { PriceImpact as PriceImpactValue } from 'hooks/usePriceImpact'
```

Comment:
This isn't a concrete value, it's a type. What about `IPriceImpact` (`I` for Interface), or `PriceImpactType`?

Reply:
thanks. names...